opus/application/static_media/js/hash.js

```diff
@@ -96,7 +159,14 @@ var o_hash = {
                     // each qtype will only have one value at a time
                     extras[slug] = [value];
                 } else {
-                    selections[slug] = value.replace("+", " ").split(",");
+                    // If value contains ", ", it's one value from string input. This will make sure "," in
+                    // slug value gets encoded in "%2C" in URL, and store as decoded "," in opus internal variables
+                    // selections.
+                    if (value.includes(", ")) {
```

Comment:
Try to look for the widget to see if it's a string instead of using includes.

Reply:
Using $(`input[name="${slug}"]`).hasClass("STRING") to look for widget with string input will work, but when opus is reload, the page will keep reloading if there is an input with "," in string field. The reason is because  in opusInitialization, the selector will return false because the widget is not rendered yet at this point. And at the same time, both selections (from getSelectionsExtrasFromHash) and opus.selections (from initFromHash) are build. After opusInitialization, load is called and selections will get updated correctly (from getSelectionsExtrasFromHash) and it will be different from opus.selections, which will cause page reload again. (This will happen over and over).

So for now, we didn't update this part and when there a "," in string input, the page will reload one time. And if there is no "," in the string field, page will not reload and search will be performed directly.